etc/profile-a-l/firefox-common.profile

```diff
@@ -48,8 +48,8 @@ notv
 protocol unix,inet,inet6,netlink
 # The below seccomp configuration still permits chroot syscall. See https://github.com/netblue30/firejail/issues/2506 for possible workarounds.
 seccomp !chroot
-# Disable tracelog, it breaks or causes major issues with many firefox based browsers, see https://github.com/netblue30/firejail/issues/1930.
-#tracelog
+# tracelog, it breaks or causes major issues with older (< v100) firefox based browsers, see https://github.com/netblue30/firejail/issues/1930.
+tracelog
```

Comment:
Todo: Which profiles include firefox-common.

Reply:
```
/etc/firejail/abrowser.profile
/etc/firejail/basilisk.profile
/etc/firejail/cachy-browser.profile
/etc/firejail/cliqz.profile
/etc/firejail/cyberfox.profile
/etc/firejail/firedragon.profile
/etc/firejail/firefox.profile
/etc/firejail/icecat.profile
/etc/firejail/librewolf.profile
/etc/firejail/palemoon.profile
/etc/firejail/thunderbird.profile
/etc/firejail/torbrowser.profile
/etc/firejail/waterfox.profile
```

Enabling it for all of them sounds risky to me. We should `ignore` it in untested profiles I guees (all except firefox/librewolf?).

Reply:
it has been tested with tor.
should be ignored for the browsers based on older firefox.